src/components/cities/CityPeople.tsx

```diff
@@ -32,7 +35,12 @@ export default function CityPeople({
         [partiesWithPersons]
     );
 
-    const orderedPersons = sortPersonsByLastName(allPeople);
+    const orderedPersons = useMemo(() => {
+        if (city?.peopleOrdering === 'partyRank') {
+            return sortPeopleByPartyRank(allPeople, partiesWithPersons, city.peopleOrdering);
+        }
+        return sortPersonsByLastName(allPeople);
+    }, [allPeople, partiesWithPersons, city?.peopleOrdering]);
```

Comment:
`sortPeopleByPartyRank` already defaults to `sortPersonsByLastName` when `peopleOrdering !== 'partyRank'`. Why not rename it to `sortPeople` and use just that without the need for extra clauses?